install.sh

```diff
@@ -0,0 +1,570 @@
+#! /bin/bash
+
+# USAGE:
+# export CONTAINER_TOOL=podman/docker
+# ./install.sh
+
+# Default container tool
+CONTAINER_TOOL=${CONTAINER_TOOL:-docker}
+
+# Check if an argument is provided
+if [[ "$#" -eq 1 ]]; then
+    if [[ "$1" == "docker" || "$1" == "podman" ]]; then
+        CONTAINER_TOOL="$1"
+    else
+        echo "Invalid argument: $1. Please use 'docker' or 'podman'."
+        exit 1
+    fi
+fi
+
+echo "===> Selected container tool: $CONTAINER_TOOL"
+
+
+# Extract the version number from pkg/config/version.go by getting everything
+# inside the quotes. Use -n to supress printing each line, and p to print the
+# modified line.
+SIGLENS_VERSION=`\
+    curl  --silent "https://api.github.com/repos/siglens/siglens/releases/latest" |
+    grep '"tag_name":' |
+    sed -E 's/.*"([^"]+)".*/\1/'`
```

Comment:
nit: this comment seems incorrect (it looks like this is the same comment from install_with_docker.sh, which is also incorrect)